docs/hooks/airflow.md

```diff
@@ -13,9 +13,11 @@ description: Airflow Hooks Reference
 Airflow Hook triggers a DAG run in an Airflow installation using [Airflow's REST API](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/post_dag_run).
 The hook run succeeds if the DAG was triggered, and fails otherwise.
 
-## Action file Airflow hook properties
+## Action File Airflow Hook Properties
```

Comment:
Not a fan of title case except at the very first level `<h1>` / `#`.  But if we _do_ do this, then I think we need to fix other places -- [l.47](https://github.com/treeverse/lakeFS/pull/6207/files#diff-4ec15e5336d2804d7ab890a6ef876bdec527b2c827dad89b087201ddd0348e71R47) is the next one in this file.

Reply:
ack. reverted.